scripts/README.md

```diff
@@ -17,12 +17,15 @@ Syntax: ./build.sh [start_cmd] [-] [end_cmd]
 
 ## How to Customize
 
-1. Copy the `default_config.sh` file to `config.sh` in the scripts directory.
+1. Copy the `chroot_files/default_config.sh` file to `chroot_files/config.sh` in the scripts directory.
```

Comment:
It's my understanding that `default_config.sh` and `config.sh` are not specific to the chroot environment.  Rather, they specify a variety of settings for how the ISO should be generated.  I think that making this doc change would imply that the configuration settings are limited to the chroot env.

Reply:
You're right, `default_config.sh` and `config.sh` should not be in `chroot_files`